### PR TITLE
2838 20 no variables created note does not reflect actual created count

### PR DESCRIPTION
**Pull request by robinhoodie0823**
### Why does this PR exist?
Current plugin only calculates the update variables which have references when creating variables.

Closes #2839  

### What does this pull request do?
Updated the calculating logic to calculate the newly created variables when creating variables.

### Additional Notes (if any)

https://github.com/tokens-studio/figma-plugin/assets/103296157/32144463-ac1f-477a-8b16-c81ebb59e8e4

### CODE REVIEW

File: packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts [hunk 1]
```diff
@@ -46,6 +50,7 @@ export default async function createLocalVariablesInPlugin(tokens: Record<string
       const allVariableObj = await updateVariables({
         collection, mode: modeId, theme, tokens, settings,
       });
+      figmaVariablesAfterCreate += allVariableObj.removedVariables.length;
```
Comment (six7) [inline packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts:53]: im not sure i understand. we add removed variables here? wouldnt we want to capture "removed variables" as a dedicated item to notify the user `5 variables removed` ?

**Review by six7 — COMMENTED** [review]
just a question on the removedVariables part, not sure i understand why we add that and not note that to the user as a specific thing

File: packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts (hunk 1 above)
Comment (robinhoodie0823) [reply packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts:53]: The reason why I added the amount of removed tokens is here to let the users know the exact number of created variables.
If user has one token and created as variables. After that, he removed that token and created a new one. In this case, if I do not include the removed variables amount here, then plugin will notify like this `No variables were created`.

Comment (six7) [reply packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts:53]: got it!

**Review by six7 — APPROVED** [review]

**Pull request merged**